src/app/donation-complete/donation-complete.component.html

```diff
@@ -60,51 +62,86 @@ <h2>Thank you!</h2>
       <p><strong>Your donation summary</strong></p>
       <p>Your donation of <strong class="figure">{{ donation.donationAmount | exactCurrency:donation.currencyCode }}</strong> has been made. That's worth <strong class="figure">{{ totalValue | exactCurrency:donation.currencyCode }}</strong> to {{ donation.charityName }}!</p>
 
-      <p *ngIf="campaign?.thankYouMessage">{{ campaign.thankYouMessage }}</p>
-    </div>
 
-    <hr>
+      <br/>
+      <p *ngIf="campaign?.thankYouMessage" id="charity-thank-you">{{ campaign.thankYouMessage }}</p>
+    </div>
 
-    <div class="receipt">
-      <p><strong>Your Receipt</strong></p>
-      <p>{{donation.createdTime | date: 'medium'}}</p>
-      <br>
-
-      <table>
-        <tr class="donationAmount">
-          <td>Your donation</td>
-          <td>{{ donation.donationAmount | exactCurrency:donation.currencyCode }}</td>
-        </tr>
-        <tr *ngIf="donation.matchedAmount > 0">
-          <td>Matched Amount</td>
-          <td>{{ donation.matchedAmount | exactCurrency:donation.currencyCode }}</td>
-        </tr>
-        <tr *ngIf="giftAidAmount > 0">
-          <td>Gift Aid Amount</td>
-          <td>{{ giftAidAmount | exactCurrency:donation.currencyCode }}</td>
-        </tr>
-        <tr *ngIf="donation.tipAmount > 0">
-          <td>Tip Amount</td>
-          <td>{{ donation.tipAmount | exactCurrency:donation.currencyCode }}</td>
-        </tr>
-        <tr>
-          <td colspan="2"><hr></td>
-        </tr>
-        <tr class="receives">
-          <td>Charity Receives</td>
-          <td>{{ totalValue | exactCurrency:donation.currencyCode }}</td>
-        </tr>
-        <tr class="total">
-          <td><strong>Total amount you have been charged</strong></td>
-          <td><strong>{{ cardChargedAmount | exactCurrency:donation.currencyCode }}</strong></td>
-        </tr>
-      </table>
+<biggive-page-section *ngIf="donationIsLarge" colourScheme="primary" primaryFullBleed="true">
+  <div id="thank-you-large-donation-wrapper">
+    <p class="thank-you-large-donation-text">
+      We are so grateful for your support. As you have donated so generously, you might be interested in finding out
+      more about becoming a match-funder.
+    </p>
+
+    <p class="thank-you-large-donation-text">
+      Our match funding partners are our Champions and make everything we do possible.
+    </p>
+
+    <div id="join-community-cta" class="button-wrapper">
+      <biggive-button
+        space-below="0"
+        colour-scheme="clear-white"
+        is-past-campaign="false"
+        is-future-campaign="false"
+        label="Find out more"
+        open-in-new-tab="true"
+        full-width="false"
+        size="large"
+        rounded="true"
+        centered="true"
+        url="https://blog.thebiggive.org.uk/funders/">
+      </biggive-button>
     </div>
+    <div style="clear: both;"></div>
```

Comment:
I think at this point I'd rather leave this as is and move on, but I realise use of floats and clear:both for this sort of thing is probably outdated style and we should be using flexbox instead since we don't need to support IE.